CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 ### Fixed
 - Visually increasing the balance after sending CCD instead of decreasing it
 - Adding newly created accounts to the address book with a blank name
+- Fix incorrect text colors in dark theme
```

Comment:
You don't have to write "Fix..." under the Fixed section. All its entries are fixes. In this case, it should be "Incorrect text colors in dark theme"